packages/ui/Scrollable/Scrollable.js

```diff
@@ -35,6 +35,7 @@ const
 		isPageDown: is('pageDown'),
 		isPageUp: is('pageUp'),
 		nop: () => {},
+		overscrollDefaultRatio: 0.5,
```

Comment:
Why did you also change here?
I think the effect of moonstone is different from sandstone. I think we don't need to change `Scrollable`.